CMakeLists.txt

```diff
@@ -557,13 +557,7 @@ if(WITH_CORE)
 
   set (WITH_QTWEBENGINE FALSE CACHE BOOL "Enable QtWebEngine support")
   if (WITH_QTWEBENGINE)
-    if(BUILD_WITH_QT6)
-      find_package(${QT_VERSION_BASE} COMPONENTS WebEngineCore REQUIRED)
-    else()
-      # QWebEnginePage is only available in widgets module in Qt5
-      find_package(${QT_VERSION_BASE} COMPONENTS WebEngineCore WebEngineWidgets REQUIRED)
-    endif()
-
+    find_package(${QT_VERSION_BASE} COMPONENTS WebEngineCore WebEngineWidgets REQUIRED)
```

Comment:
Is this change required? I was hoping to avoid the widget library when building without GUI. Could the WebEngineWidgets find package be moved to src/gui/CMakeLists.txt?